onnxruntime/test/python/transformers/test_flash_attn_cuda.py

```diff
@@ -22,6 +22,14 @@
 from onnx import TensorProto, helper
 from rotary_flash import apply_rotary_emb
 
+try:
+    from colorama import Fore, init
```

Comment:
nit: might not need add an extra dependency. I think color text can easily done by code like
```
print('\033[31m' + 'some red text' +'\033[39m')
```